emacs_anywhere.el

```diff
@@ -1,3 +1,6 @@
+(defvar ea-popup-hook nil
+  "Functions run after Emacs Anywhere creates a new frame and buffer.")
```

Comment:
It would also be cool to have the signature of the functions mentioned here.

Reply:
Thanks @Fuco1 ! 

I'm not sure about the formatting convention for this but I'll give it a shot. Incoming 👇